awx/main/utils/__init__.py

```diff
@@ -2,5 +2,8 @@
 # All Rights Reserved.
 
 # AWX
-from awx.main.utils.common import * # noqa
-from awx.main.utils.encryption import * # noqa
+from awx.main.utils.common import *  # noqa
```

Comment:
I left this one in because it's kind of nice, and this file is tiny.

Reply:
Yeah, `__init__.py` files are one of the few reasonable use-cases for star-ports